ambry-messageformat/src/main/java/com.github.ambry.messageformat/SubRecord.java

```diff
@@ -0,0 +1,8 @@
+package com.github.ambry.messageformat;
```

Comment:
Please add license in this file. Travis build failed due to following issue:
```
* What went wrong:
Execution failed for task ':ambry-messageformat:licenseMain'.
> License violations were found: /home/travis/build/linkedin/ambry/ambry-messageformat/src/main/java/com.github.ambry.messageformat/SubRecord.java}
```

Reply:
oops, thought I caught these.  Pushing change